checks.d/mongo.py

```diff
@@ -770,9 +778,16 @@ def total_seconds(td):
                     )
 
                 # Submit the metric
+                metrics_tags = (
+                    tags +
+                    [
+                        u"cluster:db:{0}".format(st),  # Keep for backward compatibilty purpose
```

Comment:
I think this justifies adding a `FIXME 6.0` in the comment, what do you think?

Also, small typo: `compatibilty` -> `compatibility`